iFixit/react-commerce
# Optimize shopify images

closes #967 

### QA

1. Visit Vercel preview
2. Verify that images are appropriately scaled (take also into account that images may be upscaled wrt expectation depending on the device pixel ratio)
